services/udp/service.go

```diff
@@ -137,8 +137,8 @@ func (s *Service) serve() {
 	defer s.wg.Done()
 
 	s.batcher.Start()
+	buf := make([]byte, UDPBufferSize)
```

Comment:
I'll be honest, I didn't. I don't know why it would cause an issue because the buffer is only used for the duration of that loop, and the size of the read is used to slice it. This can probably be left out because @sparrc looks like he's changing it in his udp-perf branch.